@plotly/dash-test-components/src/components/AsyncComponent.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
```

Comment:
Adding a set of components with simple behaviors corresponding to various usage scenarios. This component is async.